pkg/controller/jenkins/configuration/base/validate.go

```diff
@@ -60,6 +61,46 @@ func (r *ReconcileJenkinsBaseConfiguration) Validate(jenkins *v1alpha2.Jenkins)
 	return true, nil
 }
 
+func (r *ReconcileJenkinsBaseConfiguration) validateImagePullSecrets() (bool, error) {
+	var err error
+	valid := true
+	ips := r.jenkins.Spec.Master.ImagePullSecrets
+	for _, sr := range ips {
```

Comment:
`for _, sr := range ips` -> `for _, sr := range r.jenkins.Spec.Master.ImagePullSecrets`